packages/ember/tests/routing/basic_test.js

```diff
@@ -2321,6 +2336,34 @@ QUnit.test('Nested index route is not overriden by parent\'s implicit index rout
   deepEqual(router.location.path, '/posts/emberjs');
 });
 
+if (isEnabled('ember-route-serializers')) {
+  QUnit.test('Custom Route#serialize method still works [DEPRECATED]', function() {
+    Router.map(function() {
+      this.route('posts', function() {
+        this.route('index', {
+          path: ':category'
+        });
+      });
+    });
+
+    App.PostsIndexRoute = Route.extend({
+      serialize(model) {
+        return { category: model.category };
+      }
+    });
+
+    bootApplication();
+
+    run(function() {
+      expectDeprecation(function() {
+        router.transitionTo('posts', { category: 'emberjs' });
+      });
```

Comment:
Adding the deprecation message as a second argument to `expectDeprecation` will ensure that we're catching the right deprecation.